src/utils/bivariate/types/stat.types.ts

```diff
@@ -17,12 +12,14 @@ export type Step = {
   value: number;
 };
 
-/* Divisor and denominator pair */
+/* Nominator and denominator pair */
```

Comment:
also divisor?

@migrek maybe we need to have a pass on the docs on this, seems that we have confusion on the codebase?

Reply:
Currently, we use "numerator" and "denominator" terms, but we can use "divisor" together with "dividend" as an alternative (doc with terms https://kontur.fibery.io/Tasks/project/Bivariate-Manager-441).
So, the 1st option looks better to me as terms are more distinguishable.